resources/contents/blog/en/managing-time/managing-time.md

```diff
@@ -0,0 +1,102 @@
+title: "Managing time"
+date: 2019-04-08T12:13:22+02:00
+author: Rathes Sachchithananthan
+template: post
+description: Time Management is not logical but emotional, you don't manage time, you manage yourself.
+image: /images/blog/time.jpg
+categories:
+  - Life
+tags:
+  - time management
+  - productivity
+locale: en_US
+---
+
+Many people believe that time management is an effective skill they need to have to be powerful and effective, to build the life they want. Therefore they spend a lot of time investing in tools, courses and workshops to manage their time better. And yet they fail to be more effective and productive.
+
+It isn't because they are wrong about the importance of time management but it is how they see the topic and how they try to find solutions.
+
+## Time Management is not driven by logic but by emotion
+
+The truth is that there is not such a thing as time management. Not matter how you see it, time goes on, second for second. You can't manage the time. You will always have 24 hours, 1440 minutes, 86 400 seconds every single day. You can't have more time one day and less on the other, you can move some hours around, it's not possible.
+
+Logically it is impossible to manage time. And still you can manage time. But from the emotional perspective. You can't control the time but you can control how you spend your time. You can decide when you want to do things and when you want to do other things or even none at all. So time management is more self management.
+
+## You will find time if you have to
+
+There is no excuse about not having time. You can control all 86 400 seconds of your day. Let me give an example.
+
+Last year, during winter, our engineering team went out for a little field trip. It was freezing cold, still it was a wonderful trip. But when I returned I got the shock of my life.
+
+When I entered my apartment it was freezing cold, my boiler wasn't working. And when I went into the kitchen where it is located I saw a huge mess. My boiler has been leaking water the entire weekend and the kitchen was flooded.
+
+Without any further delay, I dropped my stuff and started cleaning the kitchen. I called my landlord to inform her, called the guy who was in charge of the boiler, arranged an appointment for the next morning before work and ensured that he was able to enter my apartment while I was at work. It took me like 4 to 5 hours on a Sunday but in the end I got things fixed and it wasn't too bad.
+
+If someone had called me that Sunday to ask if I had four or five hours, do you think I would have said that I had time? Definitely not. But still I had the time to do things that were totally unplanned.
+
+## Time management is about priorities
+
+I did find that time because I had to find it. If I had waited longer, it would have ended worse and have costed me a lot of money. From that point when I saw that mess it instantly became my highest priority and thus I had to make time to do things that had to be done.
```

Comment:
```suggestion
I did find that time because I had to find it. If I had waited longer, it would have ended worse and have cost me a lot of money. From that point when I saw that mess, it instantly became my highest priority and thus I had to make time to do things that had to be done. 
```